lib/analytics.js

```diff
@@ -14,16 +14,32 @@ if (process.env.REDISTOGO_URL) {
 }
 
 let analytics = {};
+let autosaveIntervalId;
```

Comment:
Nit: Call this `autoSaveIntervalId` (capital `S`) for consistency?

Reply:
I changed everything to lowercase. AFAIC autosave is one word. Changed `autoLoad` to `load` which seems to describe it better.